src/App.tsx

```diff
@@ -1,18 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
-  .map(n => `Item ${n}`);
+type Item = {
+  id: number,
+  title: string,
+};
+
+const items = getNumbers(1, 42).map(n => `Item ${n}`);
+
+const preparedItems: Item[] = items
+  .map((item, index) => ({ id: index, title: item }));
```

Comment:
It is not a good idea to use index as id. Better generate some unique id using some tool like uuid or just use value as a key.
Also, there is no need to call `map` method twice, you could do it in the first one